### PR TITLE
Initial project setup

### PR DESCRIPTION
**Pull request by wowkalucky**
Few changes to project's structure should be made.

- Due to shadowing `pylti` library name of `lti` inner app should be changed.
- Added .editorconfig file to help developers define and maintain consistent coding styles between different editors and IDEs.
- Updated secure.py example file.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# EditorConfig helps developers define and maintain
+# consistent coding styles between different editors and IDEs.
+#
+# Grab a plugin for your editor of choice here http://editorconfig.org/#download
+
+root=true                           # most-top config file in .editorconfig configuration tree
```
Comment (idegtiarov) [inline .editorconfig]: Please use not more than 2 spaces after commented line if you commenting inline.

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+[*]
+charset = utf-8
+end_of_line = lf                    # Unix-style newlines
```
Comment (idegtiarov) [inline .editorconfig]: ditto

File: bridge-adaptivity/bridge_lti/models.py [hunk 3]
```diff
@@ -19,7 +19,7 @@ class LtiParameters(models.Model):
     lis_outcome_service_url = models.CharField(max_length=300,default='')
     lis_result_sourcedid = models.CharField(max_length=300,default='')
     oauth_consumer_key = models.CharField(max_length=300,default='')
-    user_id = models.CharField(max_length=300,default='') # lti user id
+    user_id = models.CharField(max_length=300,default='') # bridge_lti user id
```
Comment (idegtiarov) [inline bridge-adaptivity/bridge_lti/models.py]: Here and in all over the file separate inline comment with two spaces from the commented line.

File: bridge-adaptivity/adaptive_edx/urls.py [hunk 4]
```diff
@@ -7,7 +7,7 @@
     url(r'^$', views.home, name='home'),
     url(r'^auth_error/', views.lti_auth_error, name='lti_auth_error'),
     url(r'^admin/', include(admin.site.urls)),
-    url(r'^lti/', include('lti.urls', namespace="lti")),
+    url(r'^lti/', include('bridge_lti.urls', namespace="bridge-lti")),
```
Comment (idegtiarov) [inline bridge-adaptivity/adaptive_edx/urls.py]: why do we need change namespace? I would leave `lti` here.

**Comment by kunanit** [comment]
Thanks! I also fixed an import statement to reflect the new app name: https://github.com/harvard-vpal/bridge-adaptivity/commit/d490b64e2f3426ae1be77511358ff49969917aab

**Comment by kunanit** [comment]
I'd be open to more project structure refactoring. Two things I have in mind are changing "adaptive_edx" to "bridge_adaptivity" to better reflect the project, and maybe changing "module" to something more descriptive.

**Pull request merged**